### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by RiotTranslateBot**
Translations update from [Weblate](https://translate.element.io/projects/element-android/element-app/) for Element Android/Element Android App.

Current translation status:

Weblate translation status

### CODE REVIEW

File: fastlane/metadata/android/cs-CZ/changelogs/40106060.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hlavní změny v této verzi: především opravy chyb.
```
Comment (ulfatufo) [inline fastlane/metadata/android/cs-CZ/changelogs/40106060.txt:1]: Hello would my parents have a good night and I want to play my head on my site so you can click to see what the screen as I'm looking for Android is still not working for me more details as to share this issue in my eyes and my mind is always on would .?@ufo

**Review by ulfatufo — APPROVED** [review]
Hello WOULD ? @MateusRodCosta ULFAT UFO

**Review by illuminated-shade — APPROVED** [review]
@No1illuminati

**Comment by bmarty** [comment]
Let's wait for the next sync

**Pull request closed without merging**